src/main/java/CarRacing/Car.java

```diff
@@ -0,0 +1,56 @@
+package CarRacing;
+
+import java.util.Random;
+
+public class Car {
+
+    private final int MAX_VALUE = 10;
+    private final int THRESHOLD = 4;
```

Comment:
상수는 private static final 로 선언해는 것을 권장하는데, 어떤 이유 때문일까요?

```suggestion
    private static final int MAX_VALUE = 10;
    private static final int THRESHOLD = 4;
```